lib/ruby_indexer/test/constant_test.rb

```diff
@@ -306,5 +306,24 @@ module A
       constant = @index["A::M"].first
       assert_instance_of(Index::Entry::Constant, constant)
     end
+
+    def test_indexing_constant_targets
+      index(<<~RUBY)
+        module A
+          B, C = [1, Y]
+          D::E, F::G = [Z, 4]
```

Comment:
Let's mix a constant and a constant path in the same multi node:

```suggestion
          D::E, F::G = [Z, 4]
          H, I::J = B, C
```